lib/declare_schema/model.rb

```diff
@@ -130,7 +128,7 @@ def belongs_to(name, scope = nil, **options)
 
         fk_options[:dependent] = options.delete(:far_end_dependent) if options.has_key?(:far_end_dependent)
 
-        if Rails::VERSION::MAJOR >= 5
+        if ActiveSupport::VERSION::MAJOR >= 5
```

Comment:
Am I understanding correctly that you're cleaning up the explicit need to `require rails` and falling back to the autoloaded `activesupport` instead?

Reply:
Yes, the tests failed on CI because rails wasn't explicitly required at this point. I glanced and found that ActiveSupport exposes the same version, so I figured we could just depend on that. Currently this entire gem still depends on `rails` because of `generators`, but not all the pieces need to.